kokoro/config/build/presubmit/rockylinux8_aarch64.gcl

```diff
@@ -0,0 +1,10 @@
+import '../common.gcl' as common
+
+config build = common.build {
+  params {
+    environment {
+      DISTRO = 'centos8'
```

Comment:
It might help to copy the [comments](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/build/presubmit/rockylinux8_x86_64.gcl#L6-L7) from the other `rockylinux8` config:
```suggestion
      // centos8 actually resolves to "rockylinux:8" in the Dockerfile:
      // https://github.com/GoogleCloudPlatform/ops-agent/blob/72488e91d2f5dbd5ba5dc8c30f493deb7802dc09/Dockerfile#L163
      DISTRO = 'centos8'
```

Reply:
Done.